### PR TITLE
chat: os1 chrome and styling updates

### PR DESCRIPTION
**Pull request by matildepark**
- Adds os1 header-bar and window borders
- Some minor commenting and refactoring to organise and clean up skeleton's classes
- Updates dark theming according to latest mocks

Desktop (light / dark) / mobile (light / dark)

Screen Shot 2020-01-29 at 1 29 36 PM
Screen Shot 2020-01-29 at 1 27 05 PM
Screen Shot 2020-01-28 at 8 08 26 PM
Screen Shot 2020-01-28 at 8 08 37 PM

### CODE REVIEW

**Comment by matildepark** [comment]
Disregard my (now-deleted) previous comments on the janky tradeoff I was making; I went back and fixed it. Couldn't live with myself shipping that.

**Comment by loganallenc** [comment]
bless you matilde!

File: pkg/interface/chat/src/js/components/skeleton.js [hunk 1]
```diff
@@ -18,36 +18,60 @@ export class Skeleton extends Component {
...
+      "w-100 inter pt4 f8": !this.props.chatHideOnMobile
+    })
+
+    // popout switches out window chrome and borders
+    let popoutWindow = this.props.popout
+    ? ""
+    : "ph4-m ph4-l ph4-xl pb4-m pb4-l pb4-xl";
+
+    let popoutBorder = this.props.popout 
+    ? ""
```
Comment (loganallenc) [inline pkg/interface/chat/src/js/components/skeleton.js]: ternary statements are fine, but should not use this style.

```let popoutWindow = this.props.popout ? "" : "asdfsdf";```

or

```
let popoutWindow = this.props.popout
  ? "" : "asdfsdfsadf";
```

or 

```
let popoutWindow = this.props.popout ? 
  (
    "asdfasdffsd" +
    "asfdkjlfsdaj"
  ) : "asdfsdfsadf";
```

any of these would be fine.
```

File: pkg/interface/chat/src/js/components/lib/header-bar.js [hunk 2]
```diff
@@ -1,36 +1,55 @@
...
+    //        style={{width: 16, height: 16, top: 16, left: 55}}>
+    //        <IconSpinner/>
+    //      </div>
+    //   :  null;
 
-      let popoutHide = (this.props.popout)
-        ? "dn dn-m dn-l dn-xl"
-        : "dn db-m db-l db-xl";
+    let popout = window.location.href.includes("popout/")
+      ? "dn"
```
Comment (loganallenc) [inline pkg/interface/chat/src/js/components/lib/header-bar.js]: again

**Pull request merged**